src/routes/api/travelRoute.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
```

Comment:
Kindly delete this file and the folder. We have changed our folder structure.

Reply:
@chuksjoe ok.. I will do just that